pkg/pool/dummypool.go

```diff
@@ -0,0 +1,98 @@
+package pool
+
+import (
+	"sync"
+
+	"github.com/pg-sharding/spqr/pkg/models/kr"
+	"github.com/pg-sharding/spqr/pkg/models/spqrerror"
+	protos "github.com/pg-sharding/spqr/pkg/protos"
+	"github.com/pg-sharding/spqr/pkg/shard"
+)
+
+type DummyPool struct {
+	Id            string
+	DB            string
+	Usr           string
+	Host          string
+	Router        string
+	ConnCount     int64
+	IdleConnCount int64
+	QueueSize     int64
```

Comment:
```
func (r *ConnectionKepperData) IdleConnectionCount() int {
	r.m.Lock()
	defer r.m.Unlock()

	return int(r.IdleConnCount)
}
```

this is how it used to look like. Where is this code? why do we rename everything and drop functionality?